src/components/Modals/Auth/LoginModal.tsx

```diff
@@ -0,0 +1,41 @@
+import OmniAural, { useOmniAural } from "omniaural"
+import { useTranslation } from "react-i18next"
+import Modal from 'react-modal'
+import { ButtonClose, TextInput } from "~/components"
+import { PV } from "~/resources"
+
+type Props = {}
+
+export const LoginModal = (props: Props) => {
+  const [modals] = useOmniAural("modals")
```

Comment:
Since you don't need this component to listen to changes to ALL to modals, I'd suggest you explicitly add the listener to observe only the "modals.login".
`const [login] = useOmniAural("modals.login")`

Reply:
good call